docs/04-api-reference/front-commerce-remix/index.mdx

```diff
@@ -31,25 +33,56 @@ the `@front-commerce/remix` reference to your `remix.env.d.ts` file.
 /// <reference types="@front-commerce/remix" />
 ```
 
-### Improved loader data types
+## Front-Commerce Context
 
-We exposed new methods in the `@front-commerce/remix` to replace the methods
-from `@remix-run` which allows for better type safety:
+To ensure the Front-Commerce Context is available in your loaders and actions,
+you need to add the `createFrontCommerceContext` to your server via the
+[`getLoadContext`](https://remix.run/docs/en/2.0.1/route/loader#context), to do
+this, update your
+[`server.ts`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/b99f0475ce494591a54fdc2b904e956a083e9b51/skeleton/server.ts)
+
+```ts
+// add-start
+import { createFrontCommerceContext } from "@front-commerce/remix";
+import { createRemixRequest } from "@remix-run/express/dist/server";
+// add-end
+
+app.all(
+  "*",
+  createRequestHandler({
+    build: require(BUILD_DIR),
+    mode: process.env.NODE_ENV,
+    // add-start
+    getLoadContext(request, response) {
+      const remixRequest = createRemixRequest(request, response);
+      return {
+        frontCommerce: createFrontCommerceContext(remixRequest),
+      };
+    },
+    // add-end
+  })
+);
+```
 
-#### `@front-commerce/remix/node`
+## `@front-commerce/remix/node`
 
 - `json`
 - `redirect`
 - `defer`
```

Comment:
I should probably add links to remix for these as well?

Reply:
Yes, that'd be great